README.md

```diff
@@ -26,15 +26,18 @@ comfort and speed.
 
 All new backend code and React components should be well tested.
 
-## Prerequisites
+## Supported operating system
 
-Houdini is designed and tested to run with the following:
+* Ubuntu 18.04, 20.04 or equivalent
+
+## Prerequisites
 
-* Ruby 2.7
 * Node 14
 * Yarn
 * PostgreSQL 10  or 12
-* Ubuntu 18.04, 20.04 or equivalent
+* Ruby 2.7
+
+> Note: All tools will be installed in the tutorial.
```

Comment:
I like how you combine the installable things together here!

We don't actually use the word "tutorial" anywhere in else in the README. Since that's the case, it would be best to use a different word that we do use, i.e. Dev Setup